### PR TITLE
Add no_params option for custom bootstrap method

### LINKED ISSUE #1475: no_params for custom bootstrap method
**Please** add support the `no_params` parameter for the custom cluster bootstrap method.

> A special no_params parameter, if defined, restricts passing parameters to custom command.

In order not to wrap the restore command to a script.

Example:
```
bootstrap:
  method: wal_g
  wal_g:
    command: wal-g backup-fetch {{ postgresql_data_dir }} LATEST
    no_params: True
    recovery_conf: 
       recovery_target_action: promote
       recovery_target_timeline: latest
       restore_command: wal-g wal-fetch %f %p
```

### PR DESCRIPTION
**Pull request by kostiantyn-nemchenko**
This PR allows setting an additional `no_params` option for custom cluster bootstrap method to prevent passing cluster-specific flags `--scope` and `--datadir`, and not wrapping WAL-G, pgBackRest etc one-liners into separate shell scripts to restore a backup.

Closes #1475

### CODE REVIEW

File: patroni/postgresql/bootstrap.py [hunk 1]
```diff
@@ -98,7 +102,8 @@ def _post_restore(self):
 
     def _custom_bootstrap(self, config):
         self._postgresql.set_state('running custom bootstrap script')
-        params = ['--scope=' + self._postgresql.scope, '--datadir=' + self._postgresql.data_dir]
+        params = [] if not self.supply_params else ['--scope=' + self._postgresql.scope,
+                                                    '--datadir=' + self._postgresql.data_dir]
```
Comment (CyberDem0n) [inline patroni/postgresql/bootstrap.py:106]: The `config` argument has everything we need:
```suggestion
        params = [] if config.get('no_params') else\
                ['--scope=' + self._postgresql.scope, '--datadir=' + self._postgresql.data_dir]
```

Any other change could go.

Comment (kostiantyn-nemchenko) [reply patroni/postgresql/bootstrap.py]: Thank you. Code updated

**Comment by Jan-M** [comment]
:+1:

**Comment by CyberDem0n** [comment]
:+1:

**Comment by CyberDem0n** [comment]
Merged, thank you @kostiantyn-nemchenko!

**Pull request merged**